infra/templates/az-isolated-service-single-region/tests/unit/unit_test.go

```diff
@@ -80,10 +79,9 @@ func TestTemplate(t *testing.T) {
 			"default_action": "Deny"
 		}]
 	}`)
-	acrNameRegex := regexp.MustCompile("\\W")
 	expectedAzureContainerRegistry := asMap(t, `{
 		"admin_enabled":       true,
-		"name":                "`+acrNameRegex.ReplaceAllString("isolated-service-"+workspace+"-azcr", "")+`",
+		"name":                "isolatedsazisolateacr",
```

Comment:
Why does this no longer need the regex?

Reply:
I decided to just hardcode what the value will be since it looked needlessly complicated when debugging the failed unit test :)